### ISSUE TITLE
Gateway licensing missing or invalid

### ISSUE THREAD

**Issue by PanYuntao** [strongloop/strong-gateway#35]
hi,
Yesterday still running,but today an error running the [demo](https://github.com/strongloop/strong-gateway) content:
···
>  StrongLoop API Gateway licensing missing or invalid. Please verify your licen
> ses in the Licenses page by running StrongLoop Arc by typing "slc arc --licenses
> " . If you have questions about your license or StrongLoop licensing please cont
> act sales@strongloop.com.
>   Terminating process.

···

**Comment by timoa** [comment]
Same problem for me... I didn't use the API Gateway module but my develop is now blocked by this error...

**Comment by raymondfeng** [comment]
@timoa Do you use `loopback-component-oauth2`?

**Comment by timoa** [comment]
Yes @raymondfeng. I saw that this morning with a dependency to strongloop-license and need a valid API Gateway license... 

Why to use OAuth2 we need to have a API Gateway license?

**Comment by joshbedo** [comment]
I'm receiving the same error, do you need a license to use the gateway now? I was under the impression that it was free.

**Comment by Iuriy-Budnikov** [comment]
I have the same issue
